projects/v2/src/app/modules/root/root.component.ts

```diff
@@ -190,7 +191,8 @@ export class RootComponent implements OnInit, OnDestroy{
 
     this.bs$.subscribe(value => {
       if (value) {
-        const bsd = store.selectSnapshot(TreeState.getBottomSheetData);
+        var bsd = store.selectSnapshot(TreeState.getBottomSheetData);
```

Comment:
1. Don't use var because they get hoisted which is bad.
2. Is there a way to not use select snapshot? Can you pass in the bottomsheetdata observable to the bottom sheet? That way we can ensure that the bottom sheet will get the data